pkg/ext-proc/backend/provider.go

```diff
@@ -217,3 +227,40 @@ func (p *Provider) refreshMetricsOnce() error {
 	}
 	return errs
 }
+
+func (p *Provider) flushPrometheusMetricsOnce() {
+	klog.V(logutil.DEFAULT).Infof("Flushing Prometheus Metrics")
```

Comment:
use DEBUG instead please, otherwise this will spam the logs

Reply:
Done.